nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/resources/META-INF/services/javax.script.ScriptEngineFactory

```diff
@@ -12,4 +12,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-org.apache.nifi.processors.script.engine.ClojureScriptEngineFactory
\ No newline at end of file
+org.apache.nifi.processors.script.engine.ClojureScriptEngineFactory
+org.jetbrains.kotlin.script.jsr223.KotlinJsr223JvmLocalScriptEngineFactory
```

Comment:
Is this necessary for the ScriptEngineFactory to be picked up by the ServiceLoader? It causes warnings when building without the profile enabled, and I'm curious as to why the ServiceLoader wouldn't pick it up if the library was included...

Reply:
I think that was a carry over from about 2 years ago when IIRC you had to add it in manually. I'll try it out without and see what happens since the newer samples don't call for this.